source/documentation/virtual_machine_management_guide/topics/Installing_Windows_on_Virtio_Optimized_Hardware.adoc

```diff
@@ -11,7 +11,7 @@ Use the *Run Once* option to attach the `virtio-win___version__.iso` file in a o
 The following items have been added to the virtual machine:
 
 * a *Red Hat VirtIO* network interface.
```

Comment:
Although you inherited the text, consider the following changes:
1. Change "have been" to "were" unless there is a reason to use the present perfect rather than the simple past.
2. Capitalize the first word of each item in the bulleted list, including articles (In the IBM Style Guide, see https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#lists )